docs/source/requirements.rst

```diff
@@ -6,7 +6,7 @@ Requirements
 * `envsubst <https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html>`_
   from the GNU gettext utilities package
 * `GNU make <https://www.gnu.org/software/make/>`_
-* `python3 <https://www.python.org/downloads/>`_
+* `python3.6 or better <https://www.python.org/downloads/>`_
```

Comment:
Shouldn't it be python 3.6 or newer? Better is subjective.

Reply:
Sure. I changed it to newer.